src/Utils/Utils.php

```diff
@@ -216,6 +228,21 @@ public static function isFindologicEnabled(SalesChannelContext $context): bool
         return $findologicService ? $findologicService->getEnabled() : false;
     }
 
+    public static function disableFindologicWhenEnabled(SalesChannelContext $context): void
```

Comment:
Why is this relevant now and not in previous SW versions?

Reply:
This caused filter on the initial page (which is only enabled by default on your local development environment), to not be clickable anymore. It also happened for previous Shopware versions, but this is pretty rare case, as it only happens when Findologic is enabled on category pages as well.